scripts/ci/Dockerfile

```diff
@@ -34,7 +38,7 @@ ENV NODE_OPTIONS="--max-old-space-size=8192"
 
 RUN yarn run build ${APP} --prod
 
-FROM $DOCKER_IMAGE_REGISTRY/docker/library/node:18-alpine3.15 as output-base
+FROM base-image as output-base
```

Comment:
No static versioning here? Can this not be dangerous?

Reply:
Ahh my bad. I misunderstood this. Somehow read it as it was not setting a specific version so that it would just auto update to latest :smile: